en-US/main.lang

```diff
@@ -670,20 +670,48 @@ Mozilla Firefox Web Browser
 
 
 # Link in the footer
+;Android Browser
+Android Browser
+
+
+# Link in the footer
+;iOS Browser
+iOS Browser
+
+
+# Link in the footer
+;Focus Browser
+Focus Browser
+
+
+# Link in the footer
+;Desktop Browser
+Desktop Browser
+
+
+# Link in the footer
+;Beta, Nightly, Developer Edition
+Beta, Nightly, Developer Edition
+
+
+# Link in the footer
 ;Download Firefox Web Browser
 Download Firefox Web Browser
 
 
+# String is obsolete. Do NOT remove.
```

Comment:
They're not obsolete yet, because they're displayed on the page. They might become later on, but it's too early too add such comment.